src/Sulu/Bundle/AdminBundle/Resources/js/containers/Form/fields/Selection.js

```diff
@@ -78,6 +78,24 @@ export default class Selection extends React.Component<Props> {
         }
     }
 
+    componentDidUpdate(prevProps: Props) {
+        this.updateListStoreSelection(prevProps);
+    }
+
+    updateListStoreSelection(prevProps: Props) {
+        if (prevProps.value === this.props.value) {
```

Comment:
Since you are using `this.props.value` below again, I would say we should destructure it.